openlibrary/templates/type/edition/view.html

```diff
@@ -1,5 +1,8 @@
 $def with (page)
 
+$ work = page.works and page.works[0] or None
+$ book_title = (work.title if work and work.get('title') else page.title or "").title()
+$ book_subtitle = work.subtitle if work and work.get('subtitle') else page.subtitle if page.get('subtitle') else None
```

Comment:
Is it sufficient to change the order? So if there's no page.subtitle at least some work.subtitle is used as a fallback?

Will change it accordingly -- if there are compelling reasons (like you describe above) where it's a value detraction to not include the work subtitle (assuming there's no edition subtitle) please post.

With in mind that we're also trying to move to a single canonical book page which shows both work and edition data on a single interface

Reply:
cc: @cdrini

Reply:
I think it should be removed; the biggest case is if the book is i.e. in French, and the French edition doesn't have a subtitle. The main idea is that title and subtitle are paired in their existence.

Ex:
```
work:
  title: The Return of the King
  subtitle: Being the Third Part of the Lord of the Rings
edition:
  title: Le Retour du roi
```

The book's page should only display the French title and *no* subtitle.

I'm not sure what the best thing to do when displayed a merged book/work page; I think it would be worthwhile to have a brainstorming session about that at some point.

Reply:
I think I would want to see...

# Le Retour du Roi
## Être la troisième partie du Seigneur des Anneaux
This book is an edition of **The Return of the King**: *Being the Third Part of the Lord of the Rings*

Reply:
For now, I am going to remove .title(), prioritize edition title and subtitle (and fallback to work title and work subtitle) and will open for discussion on Tues. Will also repost/link this in #openlibrary-g on slack

Reply:
To sync with the discussion on slack, here's a better example of why I think it's a problem: https://dev.openlibrary.org/books/OL24614768M/Die_Hamlet-Tragödie_Shakespeares

Also, I wouldn't worry about the canonical book page just yet. That's a ways off and I don't think we've really planned as to how those things will be combined. Let's make this fit our current experience first; this is a small change, and all this code will probably have to be overhauled for that feature anyways, so we can change it then.